policy/modules/apps/cdrecord.te

```diff
@@ -113,3 +132,98 @@ tunable_policy(`use_nfs_home_dirs',`
 optional_policy(`
 	resmgr_stream_connect(cdrecord_t)
 ')
+
+########################################
+#
+# Cdripper (e.g. cdda2wav) local policy
+#
+
+allow cdripper_t self:capability { ipc_lock sys_nice setgid setuid dac_override sys_rawio };
+allow cdripper_t self:process { getcap getsched setcap setrlimit setsched sigkill };
+allow cdripper_t self:fifo_file rw_fifo_file_perms;
+allow cdripper_t self:tcp_socket create_socket_perms;
+allow cdripper_t self:udp_socket create_socket_perms;
+allow cdripper_t cdripper_exec_t:file execute_no_trans;
+
+corecmd_exec_bin(cdripper_t)
+corecmd_exec_shell(cdripper_t)
+
+corenet_tcp_connect_all_unreserved_ports(cdripper_t)
```

Comment:
That seems pretty broad but ok, atleast its narrowed down to `all unreserved` only

Reply:
No longer needed, see above. ACK.

Reply:
Actually, it cannot be removed...

Reply:
But replaced with a new CDDB port definition in corenetwork.